runtime/src/bank/sysvar_cache.rs

```diff
@@ -7,7 +7,8 @@ impl Bank {
     pub(crate) fn fill_missing_sysvar_cache_entries(&self) {
         let mut sysvar_cache = self.sysvar_cache.write().unwrap();
         sysvar_cache.fill_missing_entries(|pubkey, callback| {
-            if let Some(account) = self.get_account_with_fixed_root(pubkey) {
+            let acc = self.get_account_with_fixed_root(pubkey);
```

Comment:
Why was sysvar_cache.rs changed? The new code seems equivalent, but more wordy

Reply:
There was a check here which was removed following a comment. I will revert this file then.